.gitignore

```diff
@@ -61,5 +61,6 @@ StreamingAssets*
 # Unignore specific folders
 !Plugin/external/ThirdParty/Deploy/MacOS/include
 !Plugin/external/ThirdParty/Deploy/MacOS/lib
+!Plugin/external/ThirdParty/Deploy/Windows/
```

Comment:
Why are we including all of Windows rather than just two directories like on Mac?

Reply:
again - I want to make the minimum change to get abci.sln building. I unpacked [External.7z](https://github.com/unity3d-jp/AlembicForUnity/releases/download/20180122/External.7z) into a single folder.